client/state/support/actions.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * Internal dependencies
+ */
+import {
+	SUPPORT_USER_ACTIVATED,
+	SUPPORT_USER_DEACTIVATED,
+	SUPPORT_USER_FETCH_TOKEN,
+} from 'state/action-types';
+
+import User from 'lib/user';
+
+import userSettings from 'lib/user-settings';
+
+/**
+ * Requests a support user token, then dispatches the relevant actions upon response
+ * 
+ * @param  {string} supportUser     Support username
+ * @param  {string} supportPassword Support password
+ * @return {thunk}                  The action thunk
+ */
+export function supportUserFetchToken( supportUser, supportPassword ) {
+	return ( dispatch ) => {
+		dispatch( {
+			type: SUPPORT_USER_FETCH_TOKEN,
+			supportUser
+		} );
+
+		if ( supportUser && supportPassword ) {
+			let user = new User();
```

Comment:
Even though we're not reassigning `user` in this block, my reasoning was that `user` is being mutated by the `changeUser` function so it could be confusing to use `const`